fsspec/utils.py

```diff
@@ -4,14 +4,22 @@
 import pathlib
 import re
 import sys
+import uuid
+from importlib import import_module
 from contextlib import contextmanager
 from functools import partial
-from hashlib import md5
 from urllib.parse import urlsplit
 
 DEFAULT_BLOCK_SIZE = 5 * 2 ** 20
 PY36 = sys.version_info < (3, 7)
 
+# Hashing: https://www.blake2.net/
+# 64bits perf, should be blake2s for 32bits; need be cross-plateform
+# Waiting for Blake3
+hashing, hasher = os.environ.get('PYHASH', 'hashlib.blake2b').rsplit('.', 1)
```

Comment:
Is this a standard environment variable?
It seems like this should be settable in the fsspec config too, env variable to take precedence if given, but also possible to directly override in code.